### PR TITLE
Soellingeraj/local dev test

### PR DESCRIPTION
**Pull request by free-soellingeraj**
This PR is not meant to merge, but just so I get the diff page to link for ease.

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ default: &default
 
 development:
   <<: *default
-  database: nft_development
+  database: nft_development1
```
Comment (rememberlenny) [inline config/database.yml:26]: I dont think this change should be necessary.

**Comment by rememberlenny** [comment]
I haven't been using the docker file to run this, so these changes seem they should be okay. I dont think the database name should be changed, because outside of the dockerfile that will break any existing databases with the previous name (aka mine).

**Comment by free-soellingeraj** [comment]
I agree.  I was just following the instructions on the readme
—aaron

**Comment by jrhorn424** [comment]
This change shouldn't be required. I have it working locally. @free-soellingeraj is it working for you?

**Comment by free-soellingeraj** [comment]
> This change shouldn't be required. I have it working locally. @free-soellingeraj is it working for you?

Yes, the app is working.  The db rename was not necessary.  It just said in the readme that I should rename it, but I think the readme is targeted at people starting a new app.

**Comment by jrhorn424** [comment]
@free-soellingeraj Since this isn't designed to be merged, can we tag it something useful to you and then close?

**Comment by free-soellingeraj** [comment]
Its fine just delete it.  It served its purpose

**Pull request closed without merging**